dtschema/meta-schemas/nvmem.yaml

```diff
@@ -12,6 +12,8 @@ properties:
   nvmem-cells:
     $ref: "cell.yaml#/array"
 
+  '#nvmem-cell-cells': true
+
```

Comment:
Not needed in meta-schema. What you do need is:

#nvmem-cell-cells:
  enum: [ 0, 1 ]

In nvmem.yaml in the kernel. (Really, I'd like to move those here.)

The exact constraints depends on if you want to globally define how many and what the cells contain.

Reply:
Any plans for working on this? I guess after almost 2 years not really.